DomainServices/CustomerBankInfos/CustomerBankInfoService.cs

```diff
@@ -0,0 +1,94 @@
+﻿using DomainModels;
+using DomainServices.CustomerBankInfos.Interface;
+using EntityFrameworkCore.UnitOfWork.Interfaces;
+using Infrastructure.Data.Context;
+
+namespace DomainServices.CustomerBankInfos
+{
+    public class CustomerBankInfoService : ICustomerBankInfoService
+    {
+        private readonly IUnitOfWork _unitOfWork;
+        private readonly IRepositoryFactory _repositoryFactory;
+
+        public CustomerBankInfoService(IUnitOfWork<FeatureContext> unitOfWork,
+            IRepositoryFactory<FeatureContext> repositoryFactory)
+        {
+            _unitOfWork = unitOfWork ?? throw new ArgumentNullException(nameof(unitOfWork));
+            _repositoryFactory = repositoryFactory ?? (IRepositoryFactory)_unitOfWork;
+        }
+
+        public void Deposit(long customerId, decimal amount)
+        {
+            var _customerBankInfoRepo = _unitOfWork.Repository<CustomerBankInfo>();
+
+            var query = _customerBankInfoRepo.SingleResultQuery().AndFilter(x => x.CustomerId.Equals(customerId));
+
+            var bankInfo = _customerBankInfoRepo.FirstOrDefault(query) ??
+                throw new ArgumentNullException($"User with this id: {customerId}. Do not have a bank info.");
+
+
+            if (amount <= 0)
+            {
+                throw new ArgumentException($"The deposit minimum amount must be higher than zero. Amount:{amount}");
+            }
```

Comment:
```suggestion
            if (amount <= 0)
            {
                throw new ArgumentException($"The deposit minimum amount must be higher than zero. Amount:{amount}");
            }
```
Essa validação pode ser feita diretamente no domínio